test/elixir_search_extractor/search_keyword/search_keywords_test.exs

```diff
@@ -1,10 +1,33 @@
 defmodule ElixirSearchExtractor.SearchKeyword.SearchKeywordsTest do
   use ElixirSearchExtractor.DataCase, async: true
 
-  import ElixirSearchExtractor.KeywordFixtures
+  import ElixirSearchExtractor.{AccountsFixtures, KeywordFixtures}
   alias ElixirSearchExtractor.SearchKeyword.Schemas.Keyword
   alias ElixirSearchExtractor.SearchKeyword.SearchKeywords
 
+  describe "paginated_user_keywords/2" do
+    test "returns user's keywords" do
```

Comment:
let's add a keyword that belongs to another user to make sure the only keyword that belongs to the given user is returned.